{% extends "layout.html" %} {% block title %}Project:: {% endblock %} {% block
lead %} {% endblock %} {% block page_content %}
<div class="team_sec1">
  <div class="team_col0">
    <img
      src="https://static.igem.wiki/teams/4310/wiki/team-project-banner.png"
      class="banner"
    />
  </div>
</div>
<div class="sec sec-engineering">
  <div class="sidebar-container">
    <a id="link-1" href="#sidelink-1"></a>
    <div class="sidebar-hp sidebar-engineering" id="sidebar-hp">
      <br />
      <br />
      <br />
      <a id="sidelink-1" href="#link-1"
        >Golden Gate Assembly: Prefix/Suffix Block</a
      >
      <a id="sidelink-2" href="#link-2">General Description</a>
      <a id="sidelink-3" href="#link-3">Detailed Description</a>
      <a id="sidelink-4" href="#link-4">Experiment Design</a>
      <a id="sidelink-5" href="#link-5">Experiment Protocol</a>
      <a id="sidelink-6" href="#link-6">References</a>

      <a href="#">⇪</a>
    </div>
  </div>

  <div class="container-hp-1" id="containerHp1">
    <div class="hp-text-mask">
      <span>Golden Gate Assembly: Prefix/Suffix Block</span>
    </div>
    <br />
    <div class="hp-text-mask-sub" style="padding: 0 2vw">
      <span
        >For efficient golden gate assembly, our team prepared a gene fragment
        that only contains BsaI sequences and its cleavage site so that the
        prefix and suffix could be flexibly changed and attached to both ends of
        the transcription unit, assembled using SapI. The advantage of this
        method is that it is possible to greatly increase the number of gene
        fragments that can be synthesized using the golden gate assembly at once
        without the need to use various plasmids by ordering a prefix/suffix
        fragment at a low price. According to NEB, it was confirmed that their
        high-fidelity restriction enzyme could synthesize up to 52 gene
        fragments at a time. The first BsaI prefix and the last BsaI suffix have
        EcoRI and BamHI cleavage sites for double digestion and ligation to
        pCambia 2300 vector for agrobacterium-mediated microalgae
        transformation.
      </span>
      <br />
      <br />
      <br />

      <div class="text-mask-engi">
        <span>[Figure 1] BsaI Prefix/Suffix </span>
      </div>
      <br />

      <div class="text-mask-img">
        <img
          src="https://static.igem.wiki/teams/4310/wiki/engi-img-1.png"
          alt="fr-img"
          width="70%"
        />
        <br />
      </div>
      <div class="text-mask-img">
        <img
          src="https://static.igem.wiki/teams/4310/wiki/engi-img-2.png"
          alt="fr-img"
          width="70%"
        />
        <br />
      </div>
      <div class="text-mask-img">
        <img
          src="https://static.igem.wiki/teams/4310/wiki/engi-img-3.png"
          alt="fr-img"
          width="70%"
        />
        <br />
      </div>
      <div class="text-mask-img">
        <img
          src="https://static.igem.wiki/teams/4310/wiki/engi-img-4.png"
          alt="fr-img"
          width="70%"
        />
      </div>
      <br />
      <br />
      <div class="text-mask-engi">
        <span>[Figure 2] Simple Diagram of level 1~3 assembly</span>
      </div>
      <br />
      <div class="text-mask-img">
        <img
          src="https://static.igem.wiki/teams/4310/wiki/engi-img-5.png"
          alt="fr-img"
          width="70%"
        />
      </div>
      <br />
      <a id="link-2" href="#sidelink-2"></a>

      <br />
      <br />
      <hr />

      <!-- General Description -->
      <!-- General Description -->
      <!-- General Description -->

      <div class="hp-text-mask">
        <span>General Description</span>
      </div>
      <br />
      <div class="hp-text-mask-sub" style="padding: 0 0.2vw">
        <span
          >Our team has designed and tested various biosensors for C. vulgaris,
          using inducible promoters and recombinases. Inducible promoters are
          triggered only at certain conditions, working as biosensors. Our team
          has further developed the inducible promoters into bio-switch by
          utilizing recombinases, thereby creating C. vulgaris strain that can
          constantly express certain genes once the switch is turned on. The
          detailed structure of the biosensors and switches is described below.

          <br />
          <br />

          In this project, two types of inducible promoters, pHsp70A (Kropat et
          al., 1995), pCnAFP (Kim et al., 2020), two types of recombinases,
          phiC31 (Thomson et al., 2010) and Cre (Sauer & Henderson, 1988), and a
          quorum sensing system, pTra(4X)::m35S (You et al., 2006), are tested
          in C. vulgaris. The expression of these bio-bricks has been tested in
          eukaryotic cells but has not been tested specifically for C. vulgaris.
          The purpose of this project is to validate the expression of these
          bio-bricks for C. vulgaris and to utilize them for the development of
          efficient biosensors and switches.

          <br />
          <br />

          Our research and development will not stop at biosensors and
          bio-switches. We designed a bio-switch using an inducible promoter and
          recombinase, and also designed a quorum sensing system for microalgae.
          Using these biosensors together, replacing GFP with toxin or
          antitoxin, our team’s final goal is to develop an effective
          biocontainment system for genetically modified microalgae-bacteria
          consortia that can be used in wastewater treatment systems.
        </span>
        <a id="link-3" href="#sidelink-3"></a>
        <br />
        <br />
        <br />
        <hr />

        <!-- Detailed Description for Each Biosensors -->
        <!-- Detailed Description for Each Biosensors -->
        <!-- Detailed Description for Each Biosensors -->

        <div class="hp-text-mask">
          <span>Detailed Description for Each Biosensors</span>
        </div>
        <br />
        <div class="hp-text-mask-sub" style="padding: 0 2vw">
          <span
            ><div class="engi-under">
              <p><i>Biosensor #1: Inducible Promoters (pHSP70A)</i></p>
            </div>
            In this project, two types of inducible promoters, pHsp70A (Kropat
            et al., 1995), pCnAFP (Kim et al., 2020), two types of recombinases,
            phiC31 (Thomson et al., 2010) and Cre (Sauer & Henderson, 1988), and
            a quorum sensing system, pTra(4X)::m35S (You et al., 2006), are
            tested in C. vulgaris. The expression of these bio-bricks has been
            tested in eukaryotic cells but has not been tested specifically for
            C. vulgaris. The purpose of this project is to validate the
            expression of these bio-bricks for C. vulgaris and to utilize them
            for the development of efficient biosensors and switches.

            <br />
            <br />

            Our research and development will not stop at biosensors and
            bio-switches. We designed a bio-switch using an inducible promoter
            and recombinase, and also designed a quorum sensing system for
            microalgae. Using these biosensors together, replacing GFP with
            toxin or antitoxin, our team’s final goal is to develop an effective
            biocontainment system for genetically modified microalgae-bacteria
            consortia that can be used in wastewater treatment systems.

            <br />
            <br />
            <br />

            <div class="engi-under">
              <p><i>Biosensor #1-2: Cold-inducible Promoter (pCnAFP)</i></p>
            </div>
            pCnAFP is a cold-inducible promoter derived from a polar diatom,
            Chaetoceros neogracile. The transcription initiation of a cold
            inducible promoter is activated when cells are exposed to low
            temperature, 10 °C for 2 hours (Kim et al., 2020).

            <br />
            <br />
            <br />

            <div class="engi-under">
              <p>
                <i
                  >Biosensor #2: Bioswitch: Inducible Promoter - Recombinase
                  Activity</i
                >
              </p>
            </div>
            The bio-switch functions by activating inducible promoters to
            trigger recombinases, which can irreversibly convert the genetic
            sequence. With site-specific recombinases, phiC31 and Cre, precise
            DNA cleavage and ligation are possible. Site-specific recombinases
            efficiently catalyze recombination between specific targeting sites
            to delete, insert, invert, or exchange DNA. Therefore, these groups
            of enzymes are capable of site-specific deletions, excising unwanted
            DNA from the genome (Thomson et al., 2010).

            <br />
            <br />

            The Streptomyces phage PhiC31 can site-specifically integrate DNA
            into the bacterial host by integrating a pair of specific
            recognition sites attP and attB (attachment site Phage/attachment
            site Bacteria) (Thomson et al., 2010).

            <br />
            <br />

            >A 38-kDa Cre protein, from coliphage P1, can promote genetic
            synapsis and DNA recombination in eukaryotic cells. The
            recombination takes place at a specific location known as lox. The
            recombination between two lox sites, similar to that of phiC31
            recombinase, can only be catalyzed by Cre recombinase (Sauer &
            Henderson, 1988).

            <br />
            <br />

            In this regard, the sequence between two attachment sites can be
            irreversibly deleted from the genome. This way, it is possible to
            start or stop the transcription of certain genes when needed, using
            an inducible promoter and recombinase, as shown below.

            <br />
            <br />
            <br />

            <div class="engi-under">
              <p>
                <i
                  >Biosensor #3: Quorum-Sensing Promoter & Protein
                  (Tra(4X)::m35S & VP16 TAD::TraR)
                </i>
              </p>
            </div>

            The quorum-sensing system of bacteria is tested in microalgae. In
            the case of the quorum-sensing activator, TraR, corresponding to the
            quorum-sensing-controlled promoter, pTraR, it is fused with the VP16
            protein to increase the reactivity in eukaryotic cells (You et al.,
            2006). The quorum sensing-controlled promoter is activated only when
            there is AHL produced in the presence of a bacterial signal molecule
            called N-3-oxo-octanoyl homoserine lactone (C8 AHL).

            <br />
            <br />

            In this experiment, the quorum sensing promoter (Tra(4X) :: m35S)
            and the quorum sensing protein (VP16 TAD :: TraR) are tested in C.
            vulgaris because their effects in plant cells were verified. The
            Bacterial quorum sensing promoter and protein were used after slight
            modification in a way such that they could be expressed and operated
            in eukaryotic cells. In the case of Tra Promoter, when TraR protein
            binds to C8AHL, it attaches to the Tra box
            (5′-ATGTGCAGATCTGCACAT-3′) and induces the operation of bacterial
            Tra-quorum-sensing-controlled promoter (pTraR). Therefore, the part
            from pTraR that functions as the binding site for TraR protein are
            combined with the promoter for plant cells. For the plant cell
            promoter, minimal CAMV 35S promoter (m35S) is used.

            <br />
            <br />

            The bacterial quorum sensing protein, TraR, is also modified. C8 AHL
            function as an inducer, and TraR is its receptor/regulator. The
            C-terminal region of the TraR protein is responsible for the
            transcriptional activation and binding to the Tra box of pTraR. For
            its operation in eukaryotic cells, the TraR is C-terminally fused
            with the transactivation domain of Virus Protein 16 (VP16 TAD),
            derived from Herpes simplex virus-1 (You et al., 2006). VP16
            functions for viral gene transcription and replication, promoting
            its expression after infection (Reyes et al., 2022). In this regard,
            fusing VP16 TAD to TraR can increase the reactivity of the quorum
            sensing in a eukaryotic cell.

            <br />
            <a id="link-4" href="#sidelink-4"></a>

            <br />
            <br />
            <hr />
          </span>

          <!-- Experiment Design -->
          <!-- Experiment Design -->
          <!-- Experiment Design -->

          <div class="hp-text-mask">
            <span>Experiment Design</span>
          </div>
          <br />
          <div class="hp-text-mask-sub" style="padding: 0 0.2vw">
            <span
              ><div class="engi-under">
                <p><i>Biosensor #1: Heat shock-inducible Promoter</i></p>
              </div>

              Heat shock-inducible promoter activity in C. vulgaris is tested by
              observing GFP expression. The fluorescence intensity was compared
              with control.

              <br />
              <br />

              <li>
                pHsp70A > Kozak > egfp > Tnos
                <ul>
                  a. Incubation in the dark and shifted into the light for 1
                  h.<br />
                  b. Incubation in the light and exposed to heat shock (40°C)
                  for 30 min.
                </ul>
                p35S > Kozak > eGFP > Tnos (Control)
              </li>

              <br />
              <br />
              <br />

              <div class="engi-under">
                <p><i>Biosensor #1-2: Cold-inducible Promoter</i></p>
              </div>

              Cold-inducible promoter activity in C. vulgaris is tested by
              observing GFP expression. The fluorescence intensity was compared
              with control.

              <br />
              <br />

              <li>
                pCnAFP>Kozak>eGFP>Tnos

                <ul>
                  a. Incubation in low temperature (10°C) for 2 hour.
                </ul>
                p35S>Kozak>eGFP>Tnos (Control)
              </li>

              <br />
              <br />
              <br />

              <div class="engi-under">
                <p>
                  <i
                    >Biosensor #2: Bioswitch: Inducible Promoter - Recombinase
                    Activity</i
                  >
                </p>
              </div>

              Recombinase activity, triggered by inducible promoters, is tested
              by observing GFP expression. The fluorescence intensity was
              compared with control.

              <br />
              <br />

              p35S>attB>mCherry>Tnos : pHSP70A>Kozak>phiC31>Tnos :
              attP>Kozak>eGFP>Tnos
              <br />
              (Heat shock inducible promoter + phiC31 recombinase)

              <br />

              <li>
                <ul>
                  - After recombination : p35S > attB-attP > Kozak > eGFP >
                  Tnos<br />
                  a. Incubation in the dark and shifted into the light for 1 h.
                  <br />
                  b. Incubation in the light and exposed to heat shock (40°C)
                  for 30 min.
                </ul>

                <br />
                <br />

                p35S>attB>mCherry>Tnos : pCnAFP>Kozak>phiC31>Tnos :
                attP>Kozak>eGFP>Tnos<br />
                (Cold inducible promoter + phiC31 recombinase)

                <ul>
                  - After recombination : p35S > attB-attP > Kozak > eGFP >
                  Tnos<br />
                  a. Incubation in low temperature (10°C) for 2 hours.
                </ul>

                <br />
                <br />

                p35S>loxP(1)>mCherry>Tnos : pHSP70A>Kozak>Cre>Tnos :
                loxP(2)>Kozak>eGFP>Tnos <br />
                (Heat shock inducible promoter + Cre recombinase)

                <ul>
                  - After recombination : p35S > loxP(1)-loxP(2) > Kozak > eGFP
                  > Tnos
                  <br />
                  a. Incubation in the dark and shifted into the light for 1 h.
                  <br />
                  b. Incubation in the light and exposed to heat shock (40°C)
                  for 30 min.
                </ul>

                <br />
                <br />

                p35S>loxP(1)>mCherry>Tnos : pCnAFP>Kozak>Cre>Tnos :
                loxP(2)>Kozak>eGFP>Tnos
                <br />(Cold inducible promoter + Cre recombinase)
                <ul>
                  - After recombination : p35S > loxP(1)-loxP(2) > Kozak > eGFP
                  > Tnos
                  <br />
                  a. Incubation in low temperature (10°C) for 2 hours.
                </ul>
                p35S>eGFP>nos terminator (Control)
              </li>

              <br />
              <br />
              <br />

              <div class="engi-under">
                <p><i>Biosensor #3: Quorum-Sensing Promoter & Protein</i></p>
              </div>

              Quorum-Sensing promoter activity in C. vulgaris is tested by
              observing GFP expression. The expression is observed in various
              concentrations of N-3-oxo-octanoyl homoserine lactone (C8 AHL).
              The observation is made in liquid medium with 0 μM, 0.1 μM, 1 μM,
              100μM, 1000 μM, 10000 μM C8 AHL. The fluorescence intensity was
              compared with control.

              <br />
              <br />

              p4X(TraR)::m35S>Kozak>eGFP>Tnos : pHSP70A/pRbcS2>Kozak>VP16
              TAD::TraR>Tnos
              <br />
              p4X(TraR)::m35S>Kozak>eGFP>Tnos : p35S>Kozak>VP16 TAD::TraR>Tnos
              <br />
              p35S>Kozak>eGFP>Tnos (Control)

              <a id="link-5" href="#sidelink-5"></a>
              <br />
              <br />
              <br />
              <hr />
            </span>

            <!-- Experiment Protocol -->
            <!-- Experiment Protocol -->
            <!-- Experiment Protocol -->

            <div class="hp-text-mask">
              <span>Experiment Protocol</span>
            </div>
            <br />
            <div class="hp-text-mask-sub" style="padding: 0 0.2vw">
              <div><u>Microalgae Transformation</u> (Cha et al., 2012)</div>

              <br />

              <ol>
                <li>
                  The mediums and supplements for microalgae transformation are
                  obtained from Kisan Bio, South Korea.
                </li>
              </ol>
              <ol style="font-size: 0.9vw">
                <li>
                  1. Supplement Luria Broth with 5 mM glucose, 50 mg/L kanamycin
                </li>
                <li>
                  2. Inoculate Agrobacterium. Tumefaciens (KCTC 12031) single
                  colonies in 10mL of Luria Broth
                </li>
                <li>
                  3. Under constant agitation at 200 rpm on a rotary shaker at
                  30°C, culture overnight in the dark
                </li>
                <li>
                  4. Inoculate 5mL of overnight culture from step 3 in 50mL of
                  same medium
                </li>
                <li>
                  5. Under constant agitation at 200 rpm at 27°C, culture until
                  OD600 = 0.8–1.2 in the dark
                </li>
                <li>6. Centrifuge to harvest bacterial culture</li>
                <li>
                  7. Wash with induction medium (BG11 + 100 μM acetosyringone,
                  pH 5.6) and dilute to OD600 = 0.5
                </li>
                <li>
                  8. Pre-culture total of 5 × 106 Chlorella cells from a
                  log-phase culture (OD600 = 0.5–1.0) on BG11 solidified with
                  1.5% (w/v) bacto-agar at 25°C for 5 days
                </li>
                <li>9. Harvest bacterial culture with induction medium</li>
                <li>10. Mix algal pellet with 200 μL bacterial suspension</li>
                <li>
                  11. Plate on induction medium solidified with 1.5% (w/v)
                  bacto-agar
                </li>
                <li>
                  12. Co-cultivate algae with bacteria (3 days, in dark, 25°C)
                </li>
                <li>
                  13. Harvest cells with 7mL of BG11 supplemented with 500 mg/L
                  cefotaxime and 50 mg/L kanamycin to eliminate Agrobacterium (2
                  days, in dark, 25°C)
                </li>
                <li>14. Measure eGFP expression</li>
                <li>
                  15. Plate the remaining cells on selective media with 50 mg/L
                  kanamycin and 500 mg/L cefotaxime
                </li>
                <li>
                  16. Incubate transformed cells in dark before exposure to
                  light (2 days, 25°C)
                </li>
                <li>
                  17. Grow cells on LB agar plates to detect any Agrobacterium
                  (7 days in dark, 25°C)
                </li>
              </ol>

              <br />
              <br />

              <!-- -------------------------------- -->
              <!-- -------------------------------- -->
              <!-- -------------------------------- -->

              <div>
                <u>Fluorescence microplate reader assay</u> (Molino et al.,
                2018)
              </div>

              <br />

              <ol style="font-size: 0.9vw">
                <li>
                  1. Inoculate transformed colony in 1mL BG11 selective media
                  and incubate at 25°C under constant illumination
                  (50 μmol photons/m2/s) and constant agitation at 100–150 rpm
                  on a rotary shaker, culture for 7 days
                </li>
                <li>
                  2. Transfer 100 μL cells to wells of a black, clear bottom
                  96-well plate
                </li>
                <li>
                  3. Set up microplate reader (ThermoFisher Scientific, USA) as
                  table 1
                </li>
                <li>
                  4. Measure fluorescence using BG11 medium as blank and wild
                  type cells as negative control
                </li>
                <li>
                  5. Repeat steps 5-7 with supernatant in the Deep-well plate
                  obtained by centrifugation at 3000 ×g for 10 min
                </li>
                <li>6. The fluorescent signal is normalized by OD600.</li>
              </ol>

              <br />
              <div class="text-mask-img">
                <img
                  src="https://static.igem.wiki/teams/4310/wiki/engi-img-6.png"
                  alt="fr-img"
                  width="70%"
                />
              </div>

              <br />
              <br />

              <div>
                <u>Fluorescence microscopic observation of EGFP expression</u>
                (Yang et al., 2015)
              </div>

              <ol>
                <li>
                  The mediums and supplements for microalgae transformation are
                  obtained from Kisan Bio, South Korea.
                </li>
              </ol>
              <ol style="font-size: 0.9vw">
                <li>
                  - eGFP fluorescence was visualized by using a narrow-band
                  filter (Olympus U-FBNA, excitation filter 470–495 nm, barrier
                  filter 510–550 nm).
                </li>
                <li>
                  - Chlorophyll autofluorescence was observed using a wide-band
                  filter (Olympus U-FBW, excitation filter 460–495 nm, barrier
                  filter 510 nm)
                </li>
              </ol>
              <a id="link-6" href="#sidelink-6"></a>
              <br />
              <br />
              <br />
              <hr />
            </div>

            <!-- References -->
            <!-- References -->
            <!-- References -->

            <div class="hp-text-mask">
              <span>References</span>
            </div>
            <br />
            <br />
            <div class="reference">
              <span>
                Cha, T. S., Yee, W., & Aziz, A. (2011, December 29). Assessment
                of factors affecting Agrobacterium-mediated genetic
                transformation of the unicellular green alga, Chlorella
                vulgaris. World Journal of Microbiology and Biotechnology,
                28(4), 1771–1779. https://doi.org/10.1007/s11274-011-0991-0<br /><br />

                Kim, M., Kim, J., Kim, S., & Jin, E. (2020). Heterologous Gene
                Expression System Using the Cold-Inducible CnAFP Promoter in
                Chlamydomonas reinhardtii. Journal of Microbiology and
                Biotechnology, 30(11), 1777–1784.
                https://doi.org/10.4014/jmb.2007.07024<br /><br />
                Kropat, J., von Gromoff, E. D., Müller, F. W., &amp; Beck, C. F.
                (1995). Heat shock and light activation of Achlamydomonas HSP70
                gene are mediated by independent regulatory pathways. Molecular
                and General Genetics MGG, 248(6), 727–734.
                https://doi.org/10.1007/bf02191713<br /><br />
                Molino, J. V., de Carvalho, J. C., &amp; Mayfield, S. (2018).
                Evaluation of secretion reporters to microalgae biotechnology:
                Blue to Red Fluorescent Proteins. Algal Research, 31, 252–261.
                https://doi.org/10.1016/j.algal.2018.02.018<br /><br />
                Reyes, A., Farías, M. A., Corrales, N., Tognarelli, E., &amp;
                González, P. A. (2022). Herpes simplex viruses type 1 and type
                2. Encyclopedia of Infection and Immunity, 12–36.
                https://doi.org/10.1016/b978-0-12-818731-9.00062-8<br /><br />
                Sauer, B., &amp; Henderson, N. (1988). Site-specific DNA
                recombination in mammalian cells by the cre recombinase of
                bacteriophage P1. Proceedings of the National Academy of
                Sciences, 85(14), 5166–5170.
                https://doi.org/10.1073/pnas.85.14.5166<br /><br />
                Thomson, J. G., Chan, R., Thilmony, R., Yau, Y.-Y., &amp; Ow, D.
                W. (2010). PHIC31 recombination system demonstrates heritable
                germinal transmission of site-specific excision from the
                Arabidopsis genome. BMC Biotechnology, 10(1), 17.
                https://doi.org/10.1186/1472-6750-10-17<br /><br />
                Yang, B., Liu, J., Liu, B., Sun, P., Ma, X., Jiang, Y., Wei, D.,
                &amp; Chen, F. (2015). Development of a stable genetic system
                for chlorella vulgaris—a promising green alga for CO2
                biomitigation. Algal Research, 12, 134–141.
                https://doi.org/10.1016/j.algal.2015.08.012<br /><br />
                You, Y. S., Marella, H., Zentella, R., Zhou, Y., Ulmasov, T.,
                Ho, T. H. D., & Quatrano, R. S. (2006). Use of Bacterial
                Quorum-Sensing Components to Regulate Gene Expression in Plants.
                Plant Physiology, 140(4), 1205–1212.
                https://doi.org/10.1104/pp.105.074666<br /><br />
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
{% endblock %}
